src/gluonts/evaluation/_base.py

```diff
@@ -114,6 +120,8 @@ def __init__(
         custom_eval_fn: Optional[Dict] = None,
         num_workers: Optional[int] = multiprocessing.cpu_count(),
         chunk_size: int = 32,
+        mask_invalid_timeseries: Optional[bool] = True,
```

Comment:
I think this shouldn't be `Optional`. `Optional` means that a value also can be `None`, not that the argument is optional.

Reply:
Thanks!